### PR TITLE
Add dataset Misogyny in Italian

### PR DESCRIPTION
**Pull request by dnozza**
Add Automatic Misogyny Identification in Italian

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1279,6 +1279,19 @@ https://www.aclweb.org/anthology/D17-1117)
 * Medium: Text 
 * Reference: Bosco, C., Dell'Orletta, F. and Poletto, F., 2018. Overview of the EVALITA 2018 Hate Speech Detection Task. In: EVALITA 2018-Sixth Evaluation Campaign of Natural Language Processing and Speech Tools for Italian. CEUR, pp.1-9. 
 
+#### Automatic Misogyny Identification (AMI) at Evalita 2020
+* Link to publication: [http://ceur-ws.org/Vol-2765/paper161.pdf](http://ceur-ws.org/Vol-2765/paper161.pdf)
+* Link to data: [https://github.com/dnozza/ami2020](https://github.com/dnozza/ami2020)
+* Task description: Binary (misogyny / not), Binary (aggressive / not), Binary on synthetic fairness test (misogyny / not)
+* Details of task: Sexism 
+* Size of dataset: 6,000 and 1,961 (synthetic fairness test)
+* Percentage abusive: 0.47 and 0.50 (synthetic fairness test)
```
Comment (leondz) [inline README.md:1288]: are these 0.47% and 0.50% or 47% and 50%?

Comment (dnozza) [reply README.md:1288]: 47% and 50%

Comment (leondz) [reply README.md:1288]: would you mind updating the PR to include a clarifying % symbol?

**Review by leondz — APPROVED** [review]
lgtm

**Pull request merged**